README.markdown

```diff
@@ -7793,10 +7793,70 @@ tcpsock:setoption
 
 **context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;*
 
-This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality will be implemented in future.
+This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality is implemented `v0.10.18`.
 
 This feature was first introduced in the `v0.5.0rc1` release.
 
+The `option` is a string with the option name, and the value depends on the option being set:
+
+* `keepalive`
+
+	Setting this option to true enables sending of keep-alive messages on
+	connection-oriented sockets. Make sure the `connect` function
+	had been called before, for example,
+
+```lua
+
+ tcpsock:setoption("keepalive", true)
+```
+
```

Comment:
This blank line should not exist since it stopped the current markdown bulleted list. There are similar problems below.

Reply:
accept